tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -120,4 +120,20 @@ Feature: Change data of space
       | special@@@0@@@file@@@mimeType      | image/png           |
       | special@@@0@@@id                   | %file_id%           |
       | special@@@0@@@eTag                 | %eTag%              |
-      
\ No newline at end of file
+
+
+  Scenario Outline: An admin user set no restriction quota of a Space via the Graph API
+    Given user "Alice" has created a space "Project Earth" of type "project" with quota "20"
+    When user "Alice" changes the quota of the "Project Earth" space to "<quotaValue>"
+    Then the HTTP status code should be "200"
+    When user "Alice" uploads a file inside space "Project Earth" with content "some content" to "file.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    When user "Alice" lists all available spaces via the GraphApi
+    Then the json responded should contain a space "Project Earth" with these key and value pairs:
```

Comment:
If you like you could rewrite it as

```suggestion
    When user "Alice" changes the quota of the "Project Earth" space to "<quotaValue>"
    And user "Alice" uploads a file inside space "Project Earth" with content "some content" to "file.txt" using the WebDAV API
    And user "Alice" lists all available spaces via the GraphApi
    Then the HTTP status code of responses on each endpoint should be "200, 201" respectively 
    And the json responded should contain a space "Project Earth" with these key and value pairs:
```

see https://github.com/owncloud/core/blob/47a651ca5ab617f17d5103415d0fc3ebe92df653/tests/acceptance/features/bootstrap/WebDav.php#L2556

But for that old status codes would need to be saved like in https://github.com/owncloud/core/blob/47a651ca5ab617f17d5103415d0fc3ebe92df653/tests/acceptance/features/bootstrap/WebDav.php#L2142

Reply:
I got `Expected HTTP status codes: "200, 201, 200". Found HTTP status codes: "" (Exception)`  and it may be difficult to determine where the test fell

I would try how it work in next PR.